pkg/client/k8s/k8s.go

```diff
@@ -742,7 +743,7 @@ func (k *K8sClient) ListCoreV1PVCAsRaw(opts mach_apis_meta_v1.ListOptions) (resu
 		Resource("persistentvolumeclaims").
 		VersionedParams(&opts, scheme.ParameterCodec).
 		DoRaw()
-
+	err = errors.WithStack(err)
```

Comment:
what if original err is nil?
```go
if err != nil {
  err = errors.WithStack(err)
}
```

Reply:
if error is nil WithStack will return nil